algo/src/main/java/com/hartwig/serve/dao/ServeDAO.java

```diff
@@ -141,7 +141,8 @@ private void writeActionableHotspots(@NotNull Timestamp timestamp, @NotNull List
                     ACTIONABLEHOTSPOT.APPLICABLECANCERTYPE,
                     ACTIONABLEHOTSPOT.APPLICABLEDOID,
                     ACTIONABLEHOTSPOT.BLACKLISTCANCERTYPES,
-                    ACTIONABLEHOTSPOT.DESCRIPTION,
+                    ACTIONABLEHOTSPOT.EFFICACYDESCRIPTION,
+                    ACTIONABLEHOTSPOT.EFFICACYDESCRIPTIONYEAR,
                     ACTIONABLEHOTSPOT.LEVEL,
                     ACTIONABLEHOTSPOT.EVIDENCELEVELDETAILS,
```

Comment:
Unrelated to this ticket, but it's weird that the evidence level is called "level" and the evidence level details are called "evidence level details". This is not consistent (but should have been caught in an earlier ticket!)